client/js/compute.js

```diff
@@ -379,9 +379,15 @@ const compute_ops = [
   },
   {
     name: 'in',
-    desc: 'returns true if the string x is included in array y (or property x in object y) - case sensitive',
+    desc: 'returns true if string x is included in string/array y (or property x in object y) - case sensitive',
     sample: 'var a = ${x} in ${y}',
-    call: function(v, x, y) { return v = Array.isArray(y) ? y.indexOf(x) != -1 : x in y }
+    call: function(v, x, y) { return v = Array.isArray(y) || typeof y == 'string' ? y.indexOf(x) != -1 : x in y }
+  },
+  {
+    name: 'includes',
+    desc: 'returns true if string/array x includes value y (or object x includes property y) - case sensitive',
+    sample: 'var a = ${x} includes ${y}',
+    call: function(v, x, y) { return v = Array.isArray(x) || typeof x == 'string' ? x.indexOf(y) != -1 : y in x }
   },
```

Comment:
I could not get 'in' or 'includes' to work for a property in an object.  Maybe I am doing it wrong.

Would getIndex be better to use for those cases?

Reply:
`getIndex` gets the value stored at an index. `in` checks to see if the object has a value defined at the index.

Reply:
Yes, but with getIndex can't we set the index as a property?  Then in the JSON or javascript couldn't it check to see if the value is equal to null?

I ask because I couldn't get 'in' or 'includes' to work with a property and object.  However, if someone else can get it to work, then it doesn't matter.

Reply:
I got in to work for properties and objects (that wasn't actually a problem), however, when I flipped the variables and put includes instead of in it didn't work.  I'm pretty sure it is because the 'old' includes is still in the code and needs to be deleted.

![image](https://user-images.githubusercontent.com/78324099/120419810-c2d85e00-c328-11eb-8553-ee08123ad6b6.png)